ocs_ci/templates/workloads/warp/warp_obj.yaml

```diff
@@ -0,0 +1,10 @@
+host: "s3.openshift-storage.svc"
+duration: "10m"
+concurrent: 20
+objects: 100
+obj.size: "1.5MiB"
```

Comment:
can we also have `--obj.randsize`? I see it will allow the benchmark to try out different object sizes up to  `obj.size` size.

Reply:
Yes, '--obj.randsize' can be used.  However, in this scenario I'm using object size as 1.5M which was tested and ensure there is no memory leak is seen:  https://bugzilla.redhat.com/show_bug.cgi?id=2089630#c39